tables/ro.ctb

```diff
@@ -1,126 +1,158 @@
-lowercase h 125		LATIN SMALL LETTER H
-lowercase i 24		LATIN SMALL LETTER I
-lowercase j 245		LATIN SMALL LETTER J
-lowercase k 13		LATIN SMALL LETTER K
-lowercase l 123		LATIN SMALL LETTER L
-lowercase m 134		LATIN SMALL LETTER M
-lowercase n 1345		LATIN SMALL LETTER N
-lowercase o 135		LATIN SMALL LETTER O
-lowercase p 1234		LATIN SMALL LETTER P
-lowercase q 12345		LATIN SMALL LETTER Q
-lowercase r 1235		LATIN SMALL LETTER R
-lowercase s 234		LATIN SMALL LETTER S
-lowercase t 2345		LATIN SMALL LETTER T
-lowercase u 136		LATIN SMALL LETTER U
-lowercase v 1236		LATIN SMALL LETTER V
-lowercase w 2456		LATIN SMALL LETTER W
-lowercase x 1346		LATIN SMALL LETTER X
-lowercase y 13456		LATIN SMALL LETTER Y
-lowercase z 1356		LATIN SMALL LETTER Z
-punctuation \x00a0 0		NO-BREAK SPACE
-uppercase \x00c2 1267		LATIN CAPITAL LETTER A WITH CIRCUMFLEX
-uppercase \x00ce 1467		LATIN CAPITAL LETTER I WITH CIRCUMFLEX
-lowercase \x00e2 126		LATIN SMALL LETTER A WITH CIRCUMFLEX
-lowercase \x00ee 146		LATIN SMALL LETTER I WITH CIRCUMFLEX
-uppercase \x0102 167		LATIN CAPITAL LETTER A WITH BREVE
-lowercase \x0103 16		LATIN SMALL LETTER A WITH BREVE
-uppercase \x015e 1567		LATIN CAPITAL LETTER S WITH CEDILLA
-lowercase \x015f 156		LATIN SMALL LETTER S WITH CEDILLA
-uppercase \x0162 2467		LATIN CAPITAL LETTER T WITH CEDILLA
-lowercase \x0163 246		LATIN SMALL LETTER T WITH CEDILLA
-uppercase \x0218 1567		LATIN CAPITAL LETTER S WITH COMMA BELOW
-lowercase \x0219 156		LATIN CAPITAL LETTER S WITH COMMA BELOW
-uppercase \x021a 2467		LATIN CAPITAL LETTER T WITH COMMA BELOW
-lowercase \x021b 246		LATIN SMALL LETTER T WITH COMMA BELLOW
+#
+# Copyright (C) 1995-2008 by The BRLTTY Developers.
+#
+#  This file is part of liblouis.
+#
+#  liblouis is free software: you can redistribute it and/or modify it
+#  under the terms of the GNU Lesser General Public License as
+#  published by the Free Software Foundation, either version 2.1 of the
+#  License, or (at your option) any later version.
+#
+#  liblouis is distributed in the hope that it will be useful, but
+#  WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+#  Lesser General Public License for more details.
+#
+#  You should have received a copy of the GNU Lesser General Public
+#  License along with liblouis. If not, see
+#  <http://www.gnu.org/licenses/>.
+
+# BRLTTY Text Table - Romanian
+#
+# Samuel Thibault <samuel.thibault@ens-lyon.org>
+# 
+# Updated by Stefan Moisei <vortex37 {at} gmail {dot} com>
+#
+# This table is based on the Unesco report on the progress of unification of
+# braille writing « L'ÉCRITURE BRAILLE DANS LE MONDE », by Sir Clutha
+# MACKENZIE: http://unesdoc.unesco.org/images/0013/001352/135251fo.pdf
+# The document is dated 1954, so this table may be quite outdated.
+#
+# Updated by Christian Herden <christian.herden@viewplus.com>
+# according to https://www.pharmabraille.com/braille-codes/romania-braille-code/

```

Comment:
I think it would be nice to replace this with a link to our own site, http://liblouis.org/braille-specs. Possibly we can simply link to your site.